2_base_and_sensor.ipynb

```diff
@@ -377,11 +685,55 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "Let's use the sensor information in the program. Here, we will use the information from the laser scanner.\n",
```

Comment:
If the program you are referring to has not be described yet, then it is necessary to say "a program":

    "Let's use the sensor information in a program. Here, we will use the information from the laser scanner.\n",

If the program was already described, then there is no need to edit this line.